server/db/models/Appointment.js

```diff
@@ -6,6 +6,9 @@ const schema = new Schema({
   eventId: {
     type: mongoose.Schema.Types.ObjectId,
   },
+  hostUserId: {
+    type: String, // Using string type here to prevent type conversion error since all we care if whether a given id value matches it
```

Comment:
I used string because I still want the server to respond with a "id not found" instead of "casting error", when frontend sends a invalid string like "abcd1234". The conversion is done automatically as long as the string is of 24 hex characters. 

Should I just have the route check for the string format before actually letting it hit the database?